src/index.jsx

```diff
@@ -82,97 +90,151 @@ class BerbixVerify extends React.Component {
       return baseUrl;
     }
     switch (environment) {
-      case 'sandbox':
-        return 'https://verify.sandbox.berbix.com';
-      case 'staging':
-        return 'https://verify.staging.berbix.com';
+      case "sandbox":
+        return "https://verify.sandbox.berbix.com";
+      case "staging":
+        return "https://verify.staging.berbix.com";
       default:
-        return 'https://verify.berbix.com';
+        return "https://verify.berbix.com";
     }
   }
 
   frameUrl() {
-    const { overrideUrl, version, clientId, role, templateKey, email, phone, continuation, clientToken } = this.props;
+    const {
+      overrideUrl,
+      version,
+      clientId,
+      role,
+      templateKey,
+      email,
+      phone,
+      continuation,
+      clientToken,
+      showInModal
+    } = this.props;
     if (overrideUrl != null) {
       return overrideUrl;
     }
     const token = clientToken || continuation;
     const template = templateKey || role;
-    var options = ['sdk=BerbixReact-' + SDK_VERSION];
+    var options = ["sdk=BerbixReact-" + SDK_VERSION];
     if (clientId) {
-      options.push('client_id=' + clientId);
+      options.push("client_id=" + clientId);
     }
     if (template) {
-      options.push('template=' + template);
+      options.push("template=" + template);
     }
     if (email) {
-      options.push('email=' + encodeURIComponent(email));
+      options.push("email=" + encodeURIComponent(email));
     }
     if (phone) {
-      options.push('phone=' + encodeURIComponent(phone));
+      options.push("phone=" + encodeURIComponent(phone));
     }
     if (token) {
-      options.push('client_token=' + token);
+      options.push("client_token=" + token);
     }
-    return (this.baseUrl() + '/' + version + '/verify?') + options.join('&');
+    if (showInModal) {
+      options.push("modal=true");
+    }
+    return this.baseUrl() + "/" + version + "/verify?" + options.join("&");
   }
 
   render() {
     if (!this.state.show) {
       return null;
     }
-    return (
+    const iframe = (
       <iframe
         key={this.state.idx}
         src={this.frameUrl()}
         style={{
-          backgroundColor: 'transparent',
-          border: '0 none transparent',
+          backgroundColor: "transparent",
+          border: "0 none transparent",
           padding: 0,
           margin: 0,
-          display: 'block',
-          width: '100%',
-          height: this.state.height + 'px',
-          overflow: 'hidden',
+          display: "block",
+          width: "100%",
+          height: this.state.height + "px",
+          overflow: "hidden"
         }}
         allow="camera"
         scrolling="no"
         referrerPolicy="no-referrer-when-downgrade"
       />
     );
+    if (this.props.showInModal) {
+      const disp = Math.max(this.state.height, window.innerHeight);
```

Comment:
Not sure if I follow the logic on this one - the intention is to get the height of the entire screen, whereas `this.state.height` is the height of the iframe.